README.md

```diff
@@ -78,4 +78,5 @@ But as it stands, this library should support following devices:
 - Stream Deck Neo (thanks to [@ejiektpobehuk](https://github.com/ejiektpobehuk), [@AkechiShiro](https://github.com/AkechiShiro) and [node-elgato-stream-deck](https://github.com/Julusian/node-elgato-stream-deck))
 - Ajazz AKP153 (thanks to [@ZCube](https://github.com/ZCube))
 - Ajazz AKP153E (thanks to [@teras](https://github.com/teras))
+- MiraBox HSV293S (thanks to [@czyz](https://github.com/czyz))
```

Comment:
Perhaps you could add this after the entry for the AKP815, and also amend the order of the udev rules, so that the devices remain in the same order as in the code (and as a side effect, the Ajazz brand devices and teras's contributions are kept together).